doc/src/fix_wall_stochastic.txt

```diff
@@ -0,0 +1,102 @@
+"LAMMPS WWW Site"_lws - "LAMMPS Documentation"_ld - "LAMMPS Commands"_lc :c
+
+:link(lws,http://lammps.sandia.gov)
+:link(ld,Manual.html)
+:link(lc,Commands_all.html)
+
+:line
+
+fix wall/stochastic command :h3
+
+[Syntax:]
+
+fix ID group-ID wall/stochastic reflectionstyle seed face pos temp velx vely velz accomx accomy accomz ... :pre
+
+ID, group-ID are documented in "fix"_fix.html command :ulb,l
+wall/stochastic = style name of this fix command :l
+one or more face/arg pairs may be appended :l
+reflectionstyle = diffusive or maxwell or cercignanilampis :l
+seed = random seed for stochastic wall (must be less than 900000000) :l
+face = {xlo} or {xhi} or {ylo} or {yhi} or {zlo} or {zhi} :l
+  {xlo},{ylo},{zlo} pos = EDGE or constant
+    EDGE = current lo edge of simulation box
+    constant = number like 0.0 or -30.0 (distance units)
+    temp = wall temperature
+    velx, vely, velz = wall velocity along x, y, z directions
+    accomx, accomy, accomz = accommodation coefficients along x, y, z directions (only for maxwell and cercignanilampis reflection style)
+  {xhi},{yhi},{zhi} pos = EDGE or constant or variable
+    EDGE = current hi edge of simulation box
+    constant = number like 50.0 or 100.3 (distance units)
+    temp = wall temperature
+    velx, vely, velz = wall velocity along x, y, z directions
+    accomx, accomy, accomz = accommodation coefficients along x, y, z directions (only for maxwell and cercignanilampis style) :pre
+zero or more keyword/value pairs may be appended :l
+keyword = {units} :l
+  {units} value = {lattice} or {box}
+    {lattice} = the wall position is defined in lattice units
+    {box} = the wall position is defined in simulation box units :pre
+:ule
+
+[Examples:]
+
+fix zwalls all wall/stochastic diffusive 23424 zlo EDGE 300 0.1 0.1 0 zhi EDGE 200 0.1 0.1 0
+fix ywalls all wall/stochastic maxwell 345533 ylo 5.0 300 0.1 0.0 0.0 0.8 yhi 10.0 300 0.1 0.0 0.0 0.8
+fix xwalls all wall/stochastic cercignanilampis 2308 xlo 0.0 300 0.0 0.1 0.9 0.8 0.7 xhi EDGE 300 0.0 0.1 0 0.9 0.8 0.7 units box :pre
+
+[Description:]
+
+This fix differs from the command fix wall/reflect which is equivalent

```

Comment:
The first line(s) here should explain what this fix does.  Then explain how it is different from the related fix wall/reflect.